stream/tests/test_client.py

```diff
@@ -45,7 +45,7 @@ def connect_debug():
         )
         sys.exit(1)
 
-    return stream.connect(key, secret, location="qa", timeout=30)
+    return stream.connect(key, secret, location="qa", timeout=30, use_async=use_async)
```

Comment:
as a future improvement `client` could be a fixture instead of a global variable :)